Initial BIPOC slack draft

mjang opened the pull request:
This is an **incomplete** preliminary draft, started for discussion purposes. WIP being transferred from a Google doc.

Commit:
Initial BIPOC slack draft

Commit:
delete extraneous content

autoaffection committed:
Update bipoc.rst

Commit:
Remainder of Google doc

eirinikos commented:
@mjang I'm guessing that this mini manifesto would live on its own page (linked from the [Slack page](https://www.writethedocs.org/slack/)) ... what are your thoughts? :thinking:

mjang commented:
> @mjang I'm guessing that this mini manifesto would live on its own page (linked from the [Slack page](https://www.writethedocs.org/slack/)) ... what are your thoughts? thinking

That is exactly my intent. (I'll need to add a link in the PR from that page)

mjang commented on docs/bipoc.rst:
```diff
@@ -0,0 +1,77 @@
+
+As with many other professions, BIPOC are painfully underrepresented in both technical communication and software development.
+
+<!-- SOME STATS WOULD BE USEFUL HERE. -->
```
```suggestion
Here's how you'd propose substitute text
```

eirinikos committed:
add #bipoc and #lgbtq to Slack page

This commit adds a placeholder header and placeholder for these channels.

It also cleans up some of the general info on the page.

eirinikos commented:
> > @mjang I'm guessing that this mini manifesto would live on its own page (linked from the [Slack page](https://www.writethedocs.org/slack/)) ... what are your thoughts? thinking
> 
> That is exactly my intent. (I'll need to add a link in the PR from that page)

I've opened #1280 :relaxed: 

`"Channels that serve documentarians from marginalized backgrounds"` strikes me as overly wordy .. and I think that `"marginalized"` over-emphasizes feelings of disempowerment .. any suggestions for an alternate header? :thinking:

eirinikos committed:
update BIPOC draft

This commit attempts to clean up the initial draft, which was copied from a Google Doc and represents several authors.

It also removes some areas of redundancy and adds web links for further reading.

mxsasha commented on docs/bipoc.rst:
```diff
@@ -0,0 +1,74 @@
+Our #bipoc channel
+==================
+
+This is a space dedicated to supporting BIPOC members within our community.
```
Perhaps we could explain what "BIPOC" means here? I know, but not everyone who runs into this page will know.

eirinikos replied:
I agree! One of my friends raised the same point. We'll add an explanation here or somewhere close to the top of the page.

mjang replied:
```suggestion
This is a space dedicated to supporting BIPOC (Black, Indigenous, and People of Color) members within our community.
```

mjang committed:
Merge branch 'master' into mjang-add-bipoc

mjang committed:
Update docs/bipoc.rst

mjang commented on docs/bipoc.rst:
```diff
@@ -0,0 +1,74 @@
+Our #bipoc channel
+==================
+
+This is a space dedicated to supporting BIPOC (black, Indigenous and people of color) members within our community.
```
```suggestion
This is a space dedicated to supporting BIPOC (black, indigenous and people of color) members within our community.
```

mjang committed:
Update docs/bipoc.rst

Commit:
Put article links in comments

mjang commented:
> I think these articles are too far off base.

@autoaffection I've put the links in comments (so they don't show in the build)

janine-c committed:
Adding links and tweaking wording

janine-c commented on docs/bipoc.rst:
```diff
@@ -1,29 +1,35 @@
+In a global field that is largely represented by `white`_ people, admonishing racism in spaces full of non-BIPOC people is limited in its impact. For one thing, these spaces do not capture the complexity of experiences that BIPOC have lived. BIPOC individuals and communities often experience various `intersections`_ of both privilege (e.g., `colorism`_) and discrimination (e.g., physical/cognitiv ...
 
 
-.. _white: https://nmaahc.si.edu/sites/default/files/styles/image_caption/public/images/captioned/whiteculture_info_1.png
```
The internet's a terrible place, so the infographic that used to be at this link got co-opted by racists being like "this sounds great!" - so I changed the link to go to the page where that infographic used to be.

eirinikos replied:
Thank you so much!!!! Another friend pointed this out to me and I was wondering what had happened to it ...

As a side note: I actually saved that infographic to my computer a while back. We could probably use it on the page, but I think it's better and less obtrusive to use the link ref that you updated. 👍

janine-c replied:
No problem, I'm glad I was able to catch it!

I was able to find the infographic on the Wayback Machine, so we could still make a hyperlink pretty easily through there if we wanted to. But I think the co-opting problem points to a greater issue with the infographic: if people are so indoctrinated to accept cultural white supremacy, they won't see anything wrong with what it says. So maybe it won't hurt to have that more basic information available in case people need it.

mjang committed:
Merge branch 'master' into mjang-add-bipoc

janine-c committed:
Adding line about community

In response to the Unconference conversation where we discussed the value of just hanging out.

kbatuigas commented on docs/bipoc.rst:
```diff
@@ -0,0 +1,82 @@
+* While Asians are represented at a higher rate in the tech workforce than the private sector overall, white people are 1½ times more likely than Asians to rise to an executive rank.
+* Among people of color who do enter the industry, many report isolation, discrimination and toxic work environments that prompt them to take their talent else-where. People of color leave tech at more than 3.5 times the rate of white men.
+
+To shrink racial disparities and to build equity, we offer this space as BIPOC tech writers to:
```
Made a comment in the Slack channel as well, but wanted to ask about `tech writers` vs., say, `documentarians`. The latter would be more consistent with how we refer to ourselves across WtD. _But_, to Cheryl's point from yesterday, those who aren't already in the WtD community may not be familiar with the term and want to better understand who we are and what we all do. 

Hm, I suppose we could also link out to the [Documentarians](https://www.writethedocs.org/documentarians/) page for their reference?

mjang replied:
```suggestion
To shrink racial disparities and to build equity, we offer this space as BIPOC :doc:`/documentarians`. (which includes technical writers) to:
```

mjang committed:
Update docs/bipoc.rst

Commit:
Clarifying updates, comment out notes

mjang committed:
Merge branch 'master' into mjang-add-bipoc

eirinikos commented:
> `"Channels that serve documentarians from marginalized backgrounds"` strikes me as overly wordy .. and I think that `"marginalized"` over-emphasizes feelings of disempowerment .. any suggestions for an alternate header? 🤔

Some alternate suggestions ...
* `"Channels that serve specific communities"`
* `"Channels that hold space for specific communities"`

eirinikos commented on docs/bipoc.rst:
```diff
@@ -0,0 +1,81 @@
+* While Asians are represented at a higher rate in the tech workforce than the private sector overall, white people are 1½ times more likely than Asians to rise to an executive rank.
+* Among people of color who do enter the industry, many report isolation, discrimination and toxic work environments that prompt them to take their talent else-where. People of color leave tech at more than 3.5 times the rate of white men.
+
+To shrink racial disparities and to build equity, we offer this space as BIPOC :doc:`/documentarians`. (which includes technical writers) to:
```
```suggestion
To shrink racial disparities and to build equity, we offer this space as BIPOC :doc:`/documentarians` (which includes technical writers) to:
```

I think we could remove the entire parenthetical comment. But for now, let's at least remove that period. 👍

mjang replied:
Yup, I included the parenthetical because of confusion from some on the difference between TWs and documentarians

mjang committed:
Update docs/bipoc.rst

Co-authored-by: Andrea Kao <eirinikos@gmail.com>

mjang commented:
> > `"Channels that serve documentarians from marginalized backgrounds"` strikes me as overly wordy .. and I think that `"marginalized"` over-emphasizes feelings of disempowerment .. any suggestions for an alternate header? thinking
> 
> Some alternate suggestions ...
> 
> * `"Channels that serve specific communities"`
> * `"Channels that hold space for specific communities"`

The current draft includes "marginalized" in this phrase:

"we commit ourselves to supporting communicators who have been and continue to be marginalized because of their identities."

I think that's fine. I've committed some other changes. Let's see how the build looks now (I'll add the link when the build is complete)

mjang committed:
Merge branch 'master' into mjang-add-bipoc

mjang commented:
Here's the link to the build: https://writethedocs-www--1273.org.readthedocs.build/bipoc/

As for the quote on "marginalized", I have it in comments:

> "we commit ourselves to supporting communicators who have been and continue to be marginalized because of their identities."

So you see the reference to "marginalized" doesn't show in the build, and I'm OK with that. We can discuss afterwards.

eirinikos committed:
Update slack.rst

eirinikos committed:
Merge branch 'master' into add-slack-channels

eirinikos committed:
Add doc link to new bipoc.rst page

eirinikos committed:
Merge branch 'master' into mjang-add-bipoc

eirinikos committed:
add BIPOC doc to Stay Connected table of contents

This allows people to discover more easily the existence of our BIPOC
community. Also, the BIPOC doc now has a more general title that refers
to the community, rather than the Slack channel.

eirinikos committed:
Merge branch 'master' into add-slack-channels

eirinikos committed:
Merge branch 'master' into mjang-add-bipoc

eirinikos committed:
Merge branch 'add-slack-channels' into mjang-add-bipoc

eirinikos committed:
Merge branch 'master' into mjang-add-bipoc
